lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -23,6 +23,11 @@ def add(*args, &block)
     true
   end
 
+  def reopen(logdev = nil)
+    loggers.each { |l| l.reopen(logdev) }
```

Comment:
I think this will be bad, we don't want to reopen all of the underlying loggers with the same `logdev`.